docs/en/ingest-management/elastic-agent/install-fleet-managed-agent.asciidoc

```diff
@@ -31,6 +31,8 @@ and enroll {agent} on your system.
 collect data and send it to the {stack}. Behind the scenes, {agent} runs the
 {beats} shippers or {elastic-endpoint} required for your configuration.
 
+//REVIEWERS: Is Elastic Endpoint correct here? ^^
```

Comment:
It looks correct to me. Endpoint doesn't yet support running in a container but that might be too nuanced to need documenting _here_.